src/Stryker.Core/Stryker.Core/ProjectComponents/ReadOnlyFolderComposite.cs

```diff
@@ -1,16 +1,16 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace Stryker.Core.ProjectComponents
 {
     public class ReadOnlyFolderComposite : ReadOnlyProjectComponent
     {
 
-        private readonly FolderComposite _folderComposite;
+        private readonly IParentComponent _folderComposite;
         private readonly bool _belongsToProject;
 
         public IEnumerable<IReadOnlyProjectComponent> Children => _folderComposite.Children.Select(child => child.ToReadOnlyInputComponent());
-        public ReadOnlyFolderComposite(FolderComposite folderComposite, bool belongsToProjectUnderTest) : base(folderComposite)
+        public ReadOnlyFolderComposite(IParentComponent folderComposite, bool belongsToProjectUnderTest) : base(folderComposite)
```

Comment:
This reads weird. We're receiving an IParentComponent that we know is a folderComposite somehow. And from then on we also call it a folder composite. Perhaps the IParentComponent can just be IFolderComposite? We should think about this.

Reply:
done